frontend/src/metabase/entities/actions/utils.ts

```diff
@@ -0,0 +1,18 @@
+import _ from "underscore";
+
+import type { ActionFormSettings } from "metabase-types/api/writeback";
```

Comment:
I think we should be okay importing directly from `metabase-types/api`. Would also let us escape trouble if we move things around inside the types module